packages/core/src/dialects/abstract/query-generator-typescript.ts

```diff
@@ -141,6 +141,17 @@ export class AbstractQueryGeneratorTypeScript {
     throw new Error(`removeIndexQuery has not been implemented in ${this.dialect.name}.`);
   }
 
+  /**
+   * Generates an SQL query that returns all foreign keys of a table or the foreign key constraint of a given column.
+   *
+   * @param   _tableName  The table or associated model.
```

Comment:
we haven't aligned parameters in other methods. I propose not to do it here either because we can't enforce it

Reply:
Not sure what you mean with this one, that `columnName` is not supported by all dialects?

Reply:
No, the alignment of parameters inside of the JSDoc: using multiple spaces to make them start of the same column instead of always using a single space

Reply:
Ah like that. Updated